.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: "0 3 * * MON"
+  workflow_dispatch:
+
+jobs:
+  LintAndTest:
+    strategy:
+      fail-fast: false
+      matrix:
+        crystal_version:
+          - 0.35.1
+          - 0.36.1
+          - latest
```

Comment:
As we support/deprecate Crystal versions, we simply add or remove them from this array in the future.